content_zh/docs/tasks/security/authz-tcp/index.md

```diff
@@ -41,13 +41,13 @@ keywords: [security,access-control,rbac,tcp,authorization]
     * 创建新的 `ServiceAccount` 并在**启用自动注入**的网格中部署新版本的服务：
 
         {{< text bash >}}
-        $ kubectl apply -f samples/bookinfo/platform/kube/rbac/ratings-v2-add-serviceaccount.yaml
+        $ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo-ratings-v2.yaml@
         {{< /text >}}
 
     * 创建新的 `ServiceAccount` 并在**没有启用自动注入**的网格中部署新版本的服务：
```

Comment:
this section seems duplicated?

Reply:
This one is for manual sidecar injection (`istioctl kube-inject ...`), the above one is for auto sidecar injection.